core/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 pub mod config;
 pub mod public_key;
 
-pub use config::Config;
+pub use config::{Config, keypair_from, admin_keypair_from};
+pub use ed25519_dalek::{Keypair, PublicKey, SecretKey};
```

Comment:
I understand you expose `ed25519_dalek` for signing and verifying, right?

Reply:
TBH, it would make more sense to have consumer explicitly expose `dalek-ed25519` if they need it. We can't make any guarantees about stability of `ed25519-dalek` API.